GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -224,6 +365,21 @@ public void setText(CharSequence text) {
         mIsChanged = true;
     }
 
+    public void setText(String text) {
```

Comment:
This method is unnecessary. String implements CharSequence. We can remove later.